src/qt/pivx/sendmultirow.cpp

```diff
@@ -108,9 +107,11 @@ bool SendMultiRow::addressChanged(const QString& str)
             }
         } else {
             setCssProperty(ui->lineEditAddress, "edit-primary-multi-book");
-            QString label = walletModel->getAddressTableModel()->labelForAddress(trimmedStr);
-            if (!label.isNull()){
-                ui->lineEditDescription->setText(label);
+            if (!fOnlyValidate) {
+                QString label = walletModel->getAddressTableModel()->labelForAddress(trimmedStr);
+                if (!label.isNull() && label != "") {
```

Comment:
use `label.isEmpty()`. It covers both statements.

Reply:
done